### ISSUE TITLE
How to restart a model instance

### ISSUE THREAD

**Issue by jinhuang12** [triton-inference-server/server#5983]
Hi there, I'm currently hosting multiple instances of the same model on a host and in our use case, we spawn multiple child processes within each instance. Occasionally, the child process encounters an error and either dies or goes into a zombie state. 

I want to be able to essentially restart that model instance if that occurs but I'm having trouble figuring out how to restart a model instance on Triton. I'm using the Python backend as well. 

Any help would be greatly appreciated!

**Comment by kthui** [comment]
A way to restart the instance is to [unload the model](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_model_repository.md#unload) and then [load the model](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_model_repository.md#load).

Since it is the child process of an instance dies, I think it would be better to have the instance simply restart its child process.

**Comment by kthui** [comment]
A possible workaround is to update the model file timestamp (i.e. `touch ~/model_repo/model_file/1/model.py`). Without unloading the model, load the model using the api. Triton will see the model has been updated and reload the model. The reload process will keep the previous model in the background while re-loading the model. This allows for no downtime during the reload.

> I also tried to kill the python backend stub, but tritonserver does not seem to pick up that the stub has died, is that intended?

cc @Tabrizian if this is intended?

**Comment by Tabrizian** [comment]
@jinhuang12 The server will restart the stub on the next request.

**Comment by Tabrizian** [comment]
That's true this functionality might be missing for decoupled models. @kthui could you please file a ticket for this?

**Comment by kthui** [comment]
Added the ticket DLIS-5078
